GbrainsJavaProject/app/src/main/java/com/nbgsoftware/gbrains/ui/splash/SplashFragment.java

```diff
@@ -23,10 +26,20 @@ public FragmentSplashBinding getViewBinding(LayoutInflater inflater, ViewGroup c
     }
 
     @Override
-    public SplashContract.Presenter<SplashContract.View> getPresenter() {
-        return new SplashPresenter<>();
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ActivityComponent component = getActivityComponent();
+        if (component != null) {
+            component.inject(this);
+            presenter.attachView(this);
+        }
     }
 
```

Comment:
remove getPresenter(); we should not create instance into other class, we should use dependency injection to constructor create